docs/usage.md

```diff
@@ -1 +1,248 @@
+
+## Adapter Trimming
+If specific additional trimming is required (for example, from additional tags),
+you can use any of the following command line parameters. These affect the command
+used to launch TrimGalore!
+
+### `--clip_r1 [int]`
+Instructs Trim Galore to remove bp from the 5' end of read 1 (or single-end reads).
+
+### `--clip_r2 [int]`
+Instructs Trim Galore to remove bp from the 5' end of read 2 (paired-end reads only).
+
+### `--three_prime_clip_r1 [int]`
+Instructs Trim Galore to remove bp from the 3' end of read 1 _AFTER_ adapter/quality trimming has been performed.
+
+### `--three_prime_clip_r2 [int]`
+Instructs Trim Galore to re move bp from the 3' end of read 2 _AFTER_ adapter/quality trimming has been performed.
+
+
+## Capture Kits
+
+<!-- To do -->
+
+## Job Resources
+### Automatic resubmission
+Each step in the pipeline has a default set of requirements for number of CPUs, memory and time. For most of the steps in the pipeline, if the job exits on UPPMAX with an error code of `143` (exceeded requested resources) it will automatically resubmit with higher requests (2 x original, then 3 x original). If it still fails after three times then the pipeline is stopped.
+
+### Custom resource requests
+Wherever process-specific requirements are set in the pipeline, the default value can be changed by creating a custom config file. See the files in [`conf`](../conf) for examples.
+
+## Other command line parameters
+### `--outdir`
+The output directory where the results will be saved.
+
+### `--email`
+Set this parameter to your e-mail address to get a summary e-mail with details of the run sent to you when the workflow exits. If set in your user config file (`~/.nextflow/config`) then you don't need to speicfy this on the command line for every run.
+
+### `-name`
+Name for the pipeline run. If not specified, Nextflow will automatically generate a random mnemonic.
+
+This is used in the MultiQC report (if not default) and in the summary HTML / e-mail (always).
+
+**NB:** Single hyphen (core Nextflow option)
+
+### `-resume`
+Specify this when restarting a pipeline. Nextflow will used cached results from any pipeline steps where the inputs are the same, continuing from where it got to previously.
+
+You can also supply a run name to resume a specific run: `-resume [run-name]`. Use the `nextflow log` command to show previous run names.
+
+**NB:** Single hyphen (core Nextflow option)
+
+### `-c`
+Specify the path to a specific config file (this is a core NextFlow command). Useful if using different UPPMAX
+projects or different sets of reference genomes.
+
+**NB:** Single hyphen (core Nextflow option)
+
+Note - you can use this to override defaults. For example, we run on UPPMAX but don't want to use the MultiQC
```

Comment:
This example is misleading with singularity. I think it's been updated in the RNA pipeline now.